templates/repo/editor/fork.tmpl

```diff
@@ -0,0 +1,41 @@
+{{template "base/head" .}}
+<div role="main" aria-label="{{.Title}}" class="page-content repository file editor edit">
+	{{template "repo/header" .}}
+	<div class="ui container">
+		{{template "base/alert" .}}
+		<form class="ui edit form" method="post" action="{{.RepoLink}}/_fork_to_edit/{{.BranchName | PathEscapeSegments}}">
+			{{.CsrfTokenHtml}}
+			<input type="hidden" name="page_has_posted" value="{{.PageHasPosted}}">
+			<div class="repo-editor-header">
+				<div class="ui breadcrumb field{{if .Err_TreePath}} error{{end}}">
+					<a class="section" href="{{$.BranchLink}}">{{.Repository.Name}}</a>
+					{{$n := len .TreeNames}}
+					{{$l := Eval $n "-" 1}}
+					{{range $i, $v := .TreeNames}}
+						<div class="breadcrumb-divider">/</div>
+						{{if eq $i $l}}
+							<span class="section">{{$v}}</span>
+						{{else}}
+							<span class="section"><a href="{{$.BranchLink}}/{{index $.TreePaths $i | PathEscapeSegments}}">{{$v}}</a></span>
+						{{end}}
+					{{end}}
+					<input type="hidden" id="tree_path" name="tree_path" value="{{.TreePath}}" required>
+					<input type="hidden" id="edit_operation" name="edit_operation" value="{{.EditOperation}}" required>
```

Comment:
Are these `id`s used?

Reply:
Yes, they are used in `ForkToEditFilePost` to redirect to the appropriate editing page after the fork was created.

Reply:
But `ForkToEditFilePost` only uses `name` ,  but not `id`? If `id` is used for any purpose, only frontend really needs it.

Reply:
Ah, of course. I removed them now.